bundles/org.openhab.core.thing/src/test/java/org/openhab/core/thing/binding/AbstractStorageBasedTypeProviderTest.java

```diff
@@ -75,6 +75,37 @@ public void testStateChannelTypeProperlyMappedToEntityAndBack() {
         assertThat(actual.getTags(), hasItems(expected.getTags().toArray(String[]::new)));
     }
 
+    @Test
```

Comment:
I don't think we need an extra test here. You can adjust the "old" test to a `Number:Length` with a unit of `km`. Otherwise LGTM.

Reply:
Fixed